taxonomy/algolia/client.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
```

Comment:
These files were added in https://github.com/edx/taxonomy-connector/pull/67/files#diff-16d854d1f34428bdb104335e5714e693bb434a903fbcbbc5169186d418f70dbdR1 why are they appearing in this PR as well?
maybe you need to rebase?